plugins/channelrx/localsink/localsinksettings.h

```diff
@@ -42,7 +42,7 @@ struct LocalSinkSettings
     uint32_t m_log2FFT;
     FFTWindow::Function m_fftWindow;
     bool m_reverseFilter;
-    static const uint32_t m_maxFFTBands = 20;
+    uint32_t m_maxFFTBands;
```

Comment:
why not keep it as static const ?

Reply:
As is it throws error that m_maxFFTBands couldn't be resolved.
Because "static const" member isn't definition but rather a declaration.